src/scipp/core/reduction.py

```diff
@@ -46,15 +41,12 @@ def mean(x: VariableLikeType,
         Ignore NaN's when calculating the mean.
     """
     if dim is None:
-        return _call_cpp_func(_cpp.mean, x, out=out)
+        return _call_cpp_func(_cpp.mean, x)
     else:
-        return _call_cpp_func(_cpp.mean, x, dim=dim, out=out)
+        return _call_cpp_func(_cpp.mean, x, dim=dim)
```

Comment:
Do we even need `_call_cpp_func` if there is no `out` arg?

Reply:
I don't think so. But we use it in a lot of places without an `out` arg.

Reply:
People seem to like copy-and-paste :(

Reply:
To be honest, I didn't even know what that function did. I assumed it would be a little more complicated just because the name is so generic. Should I call the functions from `_cpp` directly?

Reply:
I think so, yes.